DSCResources/MSFT_xADDomainController/MSFT_xADDomainController.psm1

```diff
@@ -164,25 +202,42 @@ function Set-TargetResource
 
 function Test-TargetResource
 {
+    [CmdletBinding()]
     [OutputType([System.Boolean])]
     param
     (
-        [Parameter(Mandatory)]
+        [Parameter(Mandatory = $true)]
+        [System.String]
```

Comment:
This is specified twice on this parameter

Reply:
I believe this is saying that line was removed and the other was added.

Reply:
I should have been more specific since it did not highlight the right line.  You have System.String and String on the same parameter:
```
        [Parameter(Mandatory = $true)]
        [System.String]
        [String]$DomainName,	        [String]$DomainName,
```

Reply:
Good catch. Missed it on all three. Fixed in next commit.